nuheat/thermostat.py

```diff
@@ -138,7 +139,10 @@ def get_data(self):
         params = {
             "serialnumber": self.serial_number
         }
-        data = self._session.request(config.THERMOSTAT_URL, params=params)
+        data = self._session.request(
+            url=get_thermostat_url(brand=self._session._brand),
```

Comment:
with my suggestion below, i believe this could be:

```suggestion
            url=self._url,
```